packages/headless/src/controllers/knowledge/generated-answer/headless-answerapi-generated-answer.test.ts

```diff
@@ -31,16 +31,30 @@ vi.mock(
   '../../../features/generated-answer/generated-answer-analytics-actions'
 );
 vi.mock('../../../features/search/search-actions');
+
 vi.mock('../../../api/knowledge/stream-answer-api', async () => {
   const originalStreamAnswerApi = await vi.importActual(
     '../../../api/knowledge/stream-answer-api'
   );
+  const queryCounter = {count: 0};
+  const queries = [
+    {q: '', requestId: ''},
+    {q: 'this est une question', requestId: '12'},
+    {q: 'this est une another question', requestId: '12'},
+    {q: '', requestId: '34'},
+    {q: 'this est une yet another question', requestId: '56'},
+  ];
   return {
     ...originalStreamAnswerApi,
     fetchAnswer: vi.fn(),
     selectAnswer: () => ({
       data: {answer: 'This est une answer', answerId: '12345_6'},
     }),
+    selectAnswerTriggerParams: () => {
+      const query = {...queries[queryCounter.count]};
+      queryCounter.count++;
+      return query;
+    },
```

Comment:
I was looking at the code, do you mind removing that empty test at line 71 ? 

  it('should be tested', () => {
    expect(true).toBe(true);
  });

Reply:
wow :grimacing: :facepalm: 

Deleted